### PR TITLE
Display message when attachments size exceeds limit

### LINKED ISSUE #2: Captured images are too large for Elis
When taking a photo, it's resolution needs to be downgraded and/or file compressed before it's sent to Elis API. With current setting, 15MB limit for documents is often exceeded

### PR DESCRIPTION
**Pull request by janmarsicek**
Relates to #2

### CODE REVIEW

File: src/components/CameraHandler/index.js [hunk 1]
```diff
@@ -200,14 +225,32 @@ class CameraHandler extends React.Component<Props, State> {
...
+        {sizeLimitExceeded && (
+          <View
+            pointerEvents="none"
+            style={{
+              ...StyleSheet.absoluteFillObject,
+              alignItems: 'center',
+              padding: 10,
+            }}
+          >
+            <Message
```
Comment (zepod) [inline src/components/CameraHandler/index.js]: Why did you decided to separate this message from the rest of message logic?

Comment (zepod) [reply src/components/CameraHandler/index.js]: I'd prefer if there was only one way of Message management

File: src/redux/modules/queues/actions.js [hunk 2]
```diff
@@ -4,12 +4,11 @@ import {
   mergeMap,
   pluck,
   catchError,
-  zip, tap,
 } from 'rxjs/operators';
 import { combineEpics, ofType } from 'redux-observable';
 import { AsyncStorage } from 'react-native';
 import { authGetJSON, errorHandler } from '../../../lib/api';
-import { apiUrl, QUEUE } from '../../../constants/config';
+import { apiUrl } from '../../../constants/config';
```
Comment (zepod) [inline src/redux/modules/queues/actions.js]: I left this there by accident. Maybe it'd be better to use the constant at line `50`. Or to remove it from `config`

File: src/components/CameraHandler/index.js [hunk 3]
```diff
@@ -200,14 +225,32 @@ class CameraHandler extends React.Component<Props, State> {
               />
             ) : <NoPermission requestPermission={this.requestPermission} />
           }
-        {currentQueueIndex !== null && !!queues && !!queues.length && (
-          <QueuePicker
-            queues={queues}
-            currentQueueIndex={currentQueueIndex}
-            onQueuePick={this.props.selectQueue}
-          />
+        {currentQueueIndex !== null
```
[the remark this answers is not in the document]
Comment (janmarsicek) [reply src/components/CameraHandler/index.js]: it doesn't fix the Flow error, I will change the type annotation of the prop

File: src/components/CameraHandler/index.js (hunk 1 above)
Comment (janmarsicek) [reply src/components/CameraHandler/index.js]: just for the record :) I wanted the message to be displayed permanently. I will move the logic out of the component and use the same way for displaying the Message as we agreed

**Pull request merged**